package/firmware/ipq-wifi/Makefile

```
@@ -189,6 +190,7 @@ $(eval $(call generate-ipq-wifi-package,edgecore_eap102,Edgecore EAP102))
 $(eval $(call generate-ipq-wifi-package,edimax_cax1800,Edimax CAX1800))
 $(eval $(call generate-ipq-wifi-package,glinet_gl-ax1800,GL.iNet GL-AX1800))
 $(eval $(call generate-ipq-wifi-package,glinet_gl-axt1800,GL.iNet GL-AXT1800))
+$(eval $(call generate-ipq-wifi-package,glinet_gl-b3000,GL.iNET B3000))
```

You will hate me for nitpicking, but the manufacturer name is not correct, it should be `GL.iNet`, as the line above:

```suggestion
$(eval $(call generate-ipq-wifi-package,glinet_gl-b3000,GL.iNet B3000))
```

I spotted 3 other times in your PR where it should be changed